carbon-design-system/carbon
# fix(styles): change form helper text mt

Closes #11905


#### Changelog

**New**

- Fix style margin-top

**Changed**

- Change the margin-top style on `#{$prefix}--form__helper-text` in the package `styles/comp/form` with `$spacing-03`


**Removed**

- $spacing-02

#### Testing / Reviewing

1. yarn build
2. yarn storybook on react package
3. Check the styles on the component `TextInput default`

![form](https://user-images.githubusercontent.com/59191719/182937946-7c5f0969-74cb-4d59-bee0-8b3ee687799e.png)

----

4. Check deploy [preview](https://deploy-preview-11909--carbon-components-react.netlify.app/?path=/story/components-textinput--default)
